src/pages/product-page/Product.jsx

```diff
@@ -1,189 +1,45 @@
-import React from "react";
+import React, { useContext } from "react";
 import { Aside, Header } from "../../components";
 import "./Product.css";
-import { tracksuit } from "../../assests/images";
+import { FilterContext } from "../../context/filter-context";
 
 export function Product() {
+  const { getSortedArr } = useContext(FilterContext);
```

Comment:
Suggestion - you can name it something more specific other than `getSortedArr`

Reply:
sure